### ISSUE TITLE
Calling `make` and `new` builtins in global scope segfaults on Linux

### ISSUE THREAD

**Issue by whereswaldon** [containous/yaegi#266]
The following program `sample.go` triggers a panic:

```go
package main

import "fmt"

var foo = make([]int, 1)

func main() {
	for _, v := range foo {
		fmt.Println(v)
	}
}
```

Expected result:
```console
$ go run ./sample.go
0
```

Got:
```console
$ yaegi ./sample.go
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0xb1 pc=0x609733]

goroutine 1 [running]:
github.com/containous/yaegi/interp.nodeType(0xc0000aa540, 0xc0000adcc0, 0xc0000dcb00, 0xc0000adb40, 0x1, 0x1497b20)
        /home/chris/Code/go/src/github.com/containous/yaegi/interp/type.go:322 +0x1cf3
github.com/containous/yaegi/interp.nodeType(0xc0000aa540, 0xc0000adcc0, 0xc0000dca00, 0x3, 0xc0001a0cb8, 0xeeab40)
        /home/chris/Code/go/src/github.com/containous/yaegi/interp/type.go:248 +0x114d
github.com/containous/yaegi/interp.(*Interpreter).gta.func1(0xc0000dc800, 0xc000187da8)
        /home/chris/Code/go/src/github.com/containous/yaegi/interp/gta.go:48 +0x532
github.com/containous/yaegi/interp.(*node).Walk(0xc0000dc800, 0xc000187da8, 0x0)
        /home/chris/Code/go/src/github.com/containous/yaegi/interp/interp.go:106 +0xc9
github.com/containous/yaegi/interp.(*node).Walk(0xc0000dc700, 0xc000187da8, 0x0)
        /home/chris/Code/go/src/github.com/containous/yaegi/interp/interp.go:110 +0x70
github.com/containous/yaegi/interp.(*node).Walk(0xc0000dc200, 0xc000187da8, 0x0)
        /home/chris/Code/go/src/github.com/containous/yaegi/interp/interp.go:110 +0x70
github.com/containous/yaegi/interp.(*Interpreter).gta(0xc0000aa540, 0xc0000dc200, 0xc0000a4668, 0x4, 0x7, 0xc0000a4668)
        /home/chris/Code/go/src/github.com/containous/yaegi/interp/gta.go:17 +0xf4
github.com/containous/yaegi/interp.(*Interpreter).Eval(0xc0000aa540, 0xc0000a2200, 0x73, 0x273, 0xc0000a2200, 0x73, 0x0, 0xc2f260)
        /home/chris/Code/go/src/github.com/containous/yaegi/interp/interp.go:240 +0x104
main.main()
        /home/chris/Code/go/src/github.com/containous/yaegi/cmd/yaegi/yaegi.go:77 +0x2e3
```

So far as I can tell, this happens whenever you try to use builtins like `make` or `new` outside of functions. You can also get this crash by running `yaegi` interactively and attempting to use `make` outside of a function definition.

Also, this is a really cool project. Thanks so much for your work on it!

**Comment by ldez** [comment]
As workaround you define explicit type:

```go
package main

import "fmt"

var foo []int = make([]int, 1)

func main() {
	for _, v := range foo {
		fmt.Println(v)
	}
}
```

**Comment by mvertes** [comment]
Fixed by #358
